spec/classes/profile/master/install_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+describe 'puppet_metrics_dashboard::profile::master::install' do
+  on_supported_os.each do |os, facts|
+    context "with facter #{RSpec.configuration.default_facter_version} on #{os}" do
+      let(:node) do
+        'testhost.example.com'
+      end
+      let(:facts) do
+        facts.merge(pe_server_version: '2019.8.6')
+      end
+
+      context 'with defaults' do
+        let(:pre_condition) do
+          <<-PRE_COND
+            service { 'pe-puppetserver': ensure => running }
+          PRE_COND
+        end
+        let(:params) { { 'manage_ldap_auth' => true } }
```

Comment:
Can we get an additional test that has `'manage_ldap_auth' => false` and validates that the `is_expected.not_to contain_package`?

Reply:
Can do, I'll try and get that added tomorrow.